staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.pb.go

```diff
@@ -5287,9 +5287,11 @@ type ContainerStatus struct {
 	// Mounts for the container.
 	Mounts []*Mount `protobuf:"bytes,14,rep,name=mounts,proto3" json:"mounts,omitempty"`
 	// Log path of container.
-	LogPath              string   `protobuf:"bytes,15,opt,name=log_path,json=logPath,proto3" json:"log_path,omitempty"`
-	XXX_NoUnkeyedLiteral struct{} `json:"-"`
-	XXX_sizecache        int32    `json:"-"`
+	LogPath string `protobuf:"bytes,15,opt,name=log_path,json=logPath,proto3" json:"log_path,omitempty"`
+	// Resource limits configuration of the container.
+	Resources            *ContainerResources `protobuf:"bytes,16,opt,name=resources,proto3" json:"resources,omitempty"`
+	XXX_NoUnkeyedLiteral struct{}            `json:"-"`
+	XXX_sizecache        int32               `json:"-"`
```

Comment:
Does this mean conatinerd api needed to be modified as well?

Reply:
fixed and merged. thanks Ruiwen. Resolving this convo.